CMakeLists.txt

```diff
@@ -12,5 +16,20 @@ add_library(win-spout MODULE
 	${win-spout_SOURCES})
 target_link_libraries(win-spout
 	libobs)
+function(copy_spout_file targetfile)
+	if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+		set(_src_suffix "x64")
+	else()
+		set(_src_suffix "win32")
+	endif()
+	add_custom_command(
+		TARGET win-spout POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy
+			"${CMAKE_SOURCE_DIR}/deps/spout/Binaries/${_src_suffix}/${targetfile}"
+			"${OBS_OUTPUT_DIR}/$<CONFIGURATION>/${OBS_PLUGIN_DESTINATION}/${targetfile}")
+endfunction()
 
 install_obs_plugin_with_data(win-spout data)
+copy_spout_file("SpoutLibrary.dll")
+copy_spout_file("SpoutLibrary.lib")
```

Comment:
Is it necessary to copu SpoutLibrary.lib to plugin folder?

Reply:
I think you're right that it's unnecessary. I think it's only used in build right?

Reply:
yes